jina/serve/streamer.py

```diff
@@ -221,3 +221,34 @@ def get_streamer():
     @staticmethod
     def _set_env_streamer_args(**kwargs):
         os.environ['JINA_STREAMER_ARGS'] = json.dumps(kwargs)
+
+
+class ExecutorStreamer:
+    def __init__(self, connection_pool: GrpcConnectionPool) -> None:
+        self._connection_pool: GrpcConnectionPool = connection_pool
+
+    def __getitem__(self, executor_name: str):
+        async def _acall_executor(
+            docs: DocumentArray,
+            request_size: int = 100,
```

Comment:
the request size is not respected, here you are not sending multiple `requests`

Reply:
decided to respect request_size